Java/TelemetrySystem/src/main/java/tddmicroexercises/telemetrysystem/services/Client.java

```diff
@@ -0,0 +1,9 @@
+package tddmicroexercises.telemetrysystem.services;
+
+public interface Client {
+    boolean getOnlineStatus();
```

Comment:
this method not necessarily is related to the client. segregate it into a different interface.